src/dataset-builder/ConceptSetCreator.test.ts

```diff
@@ -50,8 +50,8 @@ describe('ConceptSetCreator', () => {
     const rootConceptNoChildren = { ...rootConcept, hasChildren: false };
     // Act
     const user = userEvent.setup();
-    // Click the add button for the root concept.
-    await user.click(screen.getByLabelText(`add ${rootConceptNoChildren.id}`));
+    // Check the LabeledCheckbox button for the root concept.
```

Comment:
I think `Click` is still the user action here, even if clicking now checks vs turning a plus into a minus. Also there's no need to make the comment specific about the underlying control as it might change but the test would still be valid.
```suggestion
    // Click the select button for the root concept.
```